Topics/Tech_Stacks.md

```diff
@@ -1 +1,184 @@
 ## Tech Stacks
+
+## Django
```

Comment:
Please move this to a Django specific file (maybe even under Python) and use references and links freely instead of bringing the content from somewhere else into the repo. What matters is making it accessible for students in the future (and updated) rather than "writing" the content into this repo

Reply:
As you suggested, this has been moved to a Django file and the use of links and references is more widespread for future use of the content.